extensions/trace-propagators/src/test/java/io/opentelemetry/extension/trace/propagation/OtTracerPropagatorTest.java

```diff
@@ -27,6 +27,7 @@
 
   private static final TraceState TRACE_STATE_DEFAULT = TraceState.builder().build();
   private static final String TRACE_ID = "ff000000000000000000000000000041";
+  private static final String TRACE_ID_RIGHT_PART = TRACE_ID.substring(TraceId.getHexLength() / 2);
```

Comment:
In order to have this in a more explicit fashion, I'd simply go and use the hardcoded value itself, as other values do here, i.e. `TRACE_ID_RIGHT_PART = "0000000000000041"`.

Reply:
done